<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPicker</class>
 <widget class="QWidget" name="ColorPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>289</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ColorPicker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(90, 90, 90);
	color: rgb(255,255,255);
	border-width: 0px;
	border-style: solid;
	border-color: transparent;
	border-radius:5px;
}
QPushButton:disabled, 
QComboBox:disabled,
QSpinBox:disabled, 
QDoubleSpinBox:disabled {
	background-color: rgb(90, 90, 90);
	color: rgb(140,140,140);
	border-color: rgb(120,120,120)
}

QPushButton {
	border-width: 2px;
	border-color: rgb(150,150,150);
}
QLabel {
	border-color: transparent;
	color: rgb(170,170,170);
}

QGroupBox {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 rgb(110, 110, 110), stop: 1 rgb(140, 130, 120));
	margin-top: 1.5ex; /* leave space at the top for the title */
	border-radius: 16px;
}
QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top center; /* position at the top center */
	color: rgb(255, 193, 100);
}
QGraphicsView, #heightmapLabel {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
             stop:0 rgb(108, 128, 108), 
             stop:1 rgb(0, 0, 0));
	border: 2px solid gray;
	border-radius: 5px;
	border-color: rgb(120, 120, 120);
	margin-top: 1ex; /* leave space at the top for the title */
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>HSV</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>H</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="colorH">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>S</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="colorS">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>V</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="colorV">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinColorH">
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinColorS">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinColorV">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <layout class="QHBoxLayout" name="wheelTab"/>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
